ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -202,12 +202,12 @@ def map_resource(geocat_resource, issued, modified, rights):
     resource_dict['media_type'] = geocat_resource.get('media_type', '')
     name = geocat_resource.get('name')
     protocol_name = geocat_resource.get('protocol_name')
-    if name and protocol_name.startswith("Map"):
+    if name and protocol_name and protocol_name.startswith("Map"):
         resource_dict['title'] = \
-            {'de': protocol_name + geocat_resource['name']['de'],
-             'fr': protocol_name + geocat_resource['name']['fr'],
-             'en': protocol_name + geocat_resource['name']['en'],
-             'it': protocol_name + geocat_resource['name']['it']}
+            {'de': protocol_name + " " + geocat_resource['name']['de'],
+             'fr': protocol_name + " " + geocat_resource['name']['fr'],
+             'en': protocol_name + " " + geocat_resource['name']['en'],
```

Comment:
```suggestion
             'en': protocol_name + " " + (geocat_resource['name']['en'].removeprefix("Preview")),
```
Something like this to remove duplicated words in the resource Map title?

Reply:
This did not work for me: I will use `lstrip` instead.

Reply:
sure, this was just an idea. All other languages we leave like it is, right?